api/swagger.yml

```diff
@@ -3245,6 +3272,11 @@ paths:
         schema:
           type: boolean
           default: true
+      - in: query
```

Comment:
I think `ObjectStatsList` description should be modified for the double meaning of the property.

Reply:
I added a description paragraph to the field to explain the expected results